src/arguments.hpp

```diff
@@ -3,6 +3,7 @@
 
 #pragma once
 
+#include <config.hpp>
```

Comment:
I would prefer to use `#include "config. hpp"` (wrapped in double quotes) to emphasis that this file is a local file (generated at compile time).